lib/src/components/stepper/stepper.dart

```diff
@@ -0,0 +1,463 @@
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                mainAxisSize: MainAxisSize.min,
+                children: [
+                  if (subtitle != null)
+                    AnimatedDefaultTextStyle(
+                      style: ZetaTextStyles.bodyMedium.copyWith(
+                        color: _getColorForType(widget.steps[index].type),
+                      ),
+                      maxLines: 1,
+                      duration: kThemeAnimationDuration,
+                      curve: Curves.fastOutSlowIn,
+                      child: subtitle,
+                    ),
+                  AnimatedDefaultTextStyle(
+                    style: switch (widget.steps[index].type) {
+                      ZetaStepType.enabled ||
+                      ZetaStepType.complete =>
+                        ZetaTextStyles.titleLarge.copyWith(
+                          color: _colors.textDefault,
+                        ),
+                      ZetaStepType.disabled =>
+                        ZetaTextStyles.titleLarge.copyWith(
+                          color: _colors.textDisabled,
+                        ),
+                    },
+                    maxLines: 1,
+                    duration: kThemeAnimationDuration,
+                    curve: Curves.fastOutSlowIn,
+                    child: widget.steps[index].title,
+                  ),
+                ],
+              ),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _getVerticalBody(int index) {
+    return Stack(
+      children: <Widget>[
+        AnimatedCrossFade(
+          firstChild: Container(height: 0),
+          secondChild: Column(
+            children: [
+              widget.steps[index].content,
+            ],
+          ),
+          firstCurve: const Interval(0, 0.6, curve: Curves.fastOutSlowIn),
+          secondCurve: const Interval(0.4, 1, curve: Curves.fastOutSlowIn),
+          sizeCurve: Curves.fastOutSlowIn,
+          crossFadeState: _isCurrent(index)
+              ? CrossFadeState.showSecond
+              : CrossFadeState.showFirst,
+          duration: kThemeAnimationDuration,
+        ),
+      ],
+    );
+  }
+
+  Color _getColorForType(ZetaStepType type) {
+    return switch (type) {
+      ZetaStepType.complete => _colors.positive,
+      ZetaStepType.disabled => _colors.cool.shade50,
+      ZetaStepType.enabled => _colors.primary,
+    };
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return switch (widget.type) {
+      ZetaStepperType.vertical => ListView(
+          controller: widget.controller,
+          shrinkWrap: true,
+          physics: widget.physics,
+          children: [
+            for (int index = 0; index < widget.steps.length; index += 1)
+              Column(
+                key: _keys[index],
+                children: <Widget>[
+                  InkResponse(
```

Comment:
I think this InkResponse needs to be constrained - on larger screens the tappable area is too big
<img width="666" alt="image" src="https://github.com/zebrafed/zeta_flutter/assets/23633655/61ccb661-1528-4fc7-9b2f-3379d9fdfac2">